src/cljs/replumb/repl.cljs

```diff
@@ -228,7 +230,7 @@
 (def valid-opts-set
   "Set of valid option used for external input validation."
   #{:verbose :warning-as-error :target :init-fn!
-    :load-fn! :read-file-fn! :src-paths})
+    :load-fn! :read-file-fn! :src-paths :filename})
```

Comment:
this option are supposed to be user-related only...we need to find a better way to obtain what you want...